tests/utils/throttleTick.test.js

```diff
@@ -0,0 +1,77 @@
+/* global assert, suite, test */
+var throttleTick = require('utils').throttleTick;
+
+suite('utils.throttleTick', function () {
+  var ts;
+  var dts;
+  var interval = 1000;
+  var throttleTickFn = throttleTick(function (t, dt) { ts.push(t); dts.push(dt); }, interval);
+
+  test('utils.throttleTick fires callback on first tick whatever', function () {
+    ts = [];
+    dts = [];
+    assert.equal(ts.length, 0);
+    assert.equal(dts.length, 0);
+    throttleTickFn = throttleTick(function (t, dt) { ts.push(t); dts.push(dt); }, interval);
+    throttleTickFn(987634578, 12345);
```

Comment:
the specific goal was to make sure it works with arbitrary values.  declared variables to make that more clear